h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonPostgisImportExportTest.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>. H2GIS is developed by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * This code is part of the H2GIS project. H2GIS is free software; 
+ * you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details <http://www.gnu.org/licenses/>.
+ *
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+
+package org.h2gis.functions.io.geojson;
+
+import org.h2.util.StringUtils;
+import org.h2gis.api.EmptyProgressVisitor;
+import org.h2gis.postgis_jts_osgi.DataSourceFactoryImpl;
+import org.h2gis.utilities.JDBCUtilities;
+import org.junit.jupiter.api.Test;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.WKTReader;
+import org.postgresql.util.PSQLException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.ConnectException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test with PostgreSQL-PostGIS for GitHub actions
+ */
+public class GeojsonPostgisImportExportTest {
```

Comment:
Thanks. Could you please add this test in GeojsonImportExportTest class like this 
https://github.com/orbisgis/h2gis/blob/master/h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java#L1158

Maybe the existing test is enough ?

Reply:
Its because it is not the same connection than h2gis (so no tearup teardown)

Reply:
and use the same db properties because we have an action to load a postgres db
THanks a lot